docs/tutorial.rst

```diff
@@ -25,22 +25,47 @@ and send you a fun picture to cheer you up if you are sad.
 1. Create a New Project
 ^^^^^^^^^^^^^^^^^^^^^^^
 
+The first step is to create a new Rasa project. To do this, run:
 
 .. runnable::
    :description: stack-init
 
    rasa init --no-prompt
 
+The ``rasa init`` command creates the files that a Rasa project needs. 
+If you leave out the ``--no-prompt`` flag you will be asked some questions about
+how you want your project to be set up.
 
-Let's take a look at the files this created:
+This creates the following files:
 
+.. code:: bash
+
+   .
+   ├── __init__.py
+   ├── actions.py
+   ├── config.yml
+   ├── credentials.yml
+   ├── data
+   │   ├── nlu.md
+   │   └── stories.md
+   ├── domain.yml
+   ├── endpoints.yml
+   └── models
+       └── <timestamp>.tar.gz
+
+
+The most important ones are your NLU training data (``data/nlu.md``)
```

Comment:
I think highlighting the most important ones works well, but I still think that explaining what each of these files mean would be best, because with rasa init these files are given by default instead of being optional, that's why the user shouldn't wonder what they are for and what to do with them